src/style.css

```diff
@@ -0,0 +1,72 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+body {
+  font-family: Avenir, sans-serif;
+}
+
+.searchInput {
+  position: relative;
+  display: block;
+  margin-top: 20px;
+}
+
+.icon {
+  position: absolute;
+  display: block;
+  left: 26px;
+  top: 25px;
+  width: 19px;
+  height: 19px;
+}
+
+.smallIcon {
+  left: 13px;
+  top: 15px;
+  width: 11px;
+  height: 11px;
+}
+
+.input {
+  font-family: inherit;
+  font-size: 16px;
+  line-height: 22px;
+  box-sizing: border-box;
+  width: 100%;
+  height: 70px;
+  padding-left: 62px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  color: #3d4e61;
+}
```

Comment:
let's use this advice https://codeguide.academy/html-css.html#css-order

Reply:
done